packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -322,25 +323,16 @@ export async function getParameterJson(ctx: SolutionContext) {
 
   const parameterFileName = parameterFileNameTemplate.replace("@envName", ctx.envInfo.envName);
   const parameterFolderPath = path.join(ctx.root, configsFolder);
-  const parameterEnvFilePath = path.join(
-    path.join(ctx.root, configsFolder),
-    parameterFileDefaultTemplate
-  );
-
   const parameterFilePath = path.join(parameterFolderPath, parameterFileName);
-  let createNewParameterFile = false;
   try {
     await fs.stat(parameterFilePath);
   } catch (err) {
-    ctx.logProvider?.info(
-      `[${PluginDisplayName.Solution}] ${parameterFilePath} does not exist. Generate it using ${parameterEnvFilePath}.`
-    );
-    createNewParameterFile = true;
+    ctx.logProvider?.error(`[${PluginDisplayName.Solution}] ${parameterFilePath} does not exist.`);
```

Comment:
should `err` be returned here since you don't handle it anymore?

Reply:
sure, throw this error